src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -23,27 +24,34 @@
     private final AddressBook addressBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
+
     private final FilteredList<Expense> filteredExpenses;
+    private final ExpenseBook expenseBook;
+
+    private Expense expenseToBeViewed;
 
     /**
      * Initializes a ModelManager with the given addressBook and userPrefs.
      */
-    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) {
+    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyExpenseBook expenseBook, ReadOnlyUserPrefs userPrefs) {
```

Comment:
Did you accidentally edit modelManager?

Reply:
Ohh I forgot to change back to the original version;( I'll change now!